Orleans.runsettings

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+  <RunConfiguration>
+    <TargetPlatform>x64</TargetPlatform>
+    <TargetFrameworkVersion>Framework45</TargetFrameworkVersion>
```

Comment:
Why `Framework45`?

Reply:
I asked the same thing, and there is not much documentation on it. But can we just not specify it then? Supposedly all of these settings have defaults. You are not required to provide a value for a setting you don't care, which *I believe* is our case, right? or was it added by design?